desc/integrals/quad_utils.py

```diff
@@ -0,0 +1,246 @@
+"""Utilities for quadratures."""
+
+from orthax.legendre import legder, legval
+
+from desc.backend import eigh_tridiagonal, jnp, put
+from desc.utils import errorif
+
+
+def bijection_to_disc(x, a, b):
+    """[a, b] ∋ x ↦ y ∈ [−1, 1]."""
+    y = 2.0 * (x - a) / (b - a) - 1.0
+    return y
+
+
+def bijection_from_disc(x, a, b):
+    """[−1, 1] ∋ x ↦ y ∈ [a, b]."""
+    y = 0.5 * (b - a) * (x + 1.0) + a
+    return y
+
+
+def grad_bijection_from_disc(a, b):
+    """Gradient of affine bijection from disc."""
```

Comment:
Which disc? A unit disc? An open disc?

Reply:
I changed description to `"""Gradient wrt ``x`` of ``bijection_from_disc``."""` If you have better names for the function feel free to change